queue/src/main/java/uk/gov/service/payments/commons/queue/sqs/SqsQueueService.java

```diff
@@ -33,17 +34,25 @@ public SqsQueueService(AmazonSQS sqsClient, int messageMaximumWaitTimeInSeconds,
     }
 
     public QueueMessage sendMessage(String queueUrl, String messageBody) throws QueueException {
+        SendMessageRequest sendMessageRequest = new SendMessageRequest(queueUrl, messageBody);
+        return sendMessage(sendMessageRequest);
+    }
+
+    public QueueMessage sendMessage(String queueUrl, String messageBody, int delayInSeconds) throws QueueException {
+        SendMessageRequest sendMessageRequest = new SendMessageRequest(queueUrl, messageBody).withDelaySeconds(delayInSeconds);
+        return sendMessage(sendMessageRequest);
+    }
```

Comment:
Missing blank line between the methods

Reply:
well spotted! fixed.

Reply:
did you push that change?

Reply:
yes... i can see it there in the latest commit - let me know if I should have done it differently

Reply:
Ah you added in another missing blank line, but there's still one missing after line 44